packages/mobile/src/screens/user-list-screen/UserList.tsx

```diff
@@ -47,16 +59,26 @@ export const UserList = (props: UserListProps) => {
     [usersMap, userIds]
   )
 
+  useEffect(() => {
+    if (users.length !== 0) {
+      cachedUsers.current = users
+    }
+  }, [users])
+
   const handleEndReached = useCallback(() => {
     if (hasMore) {
       dispatchWeb(loadMore(tag))
     }
   }, [hasMore, dispatchWeb, tag])
 
+  if (loading && cachedUsers.current.length === 0) {
+    return <LoadingSpinner style={styles.spinner} />
```

Comment:
ty. does the loading spinner show at the bottom now too? i think that might also be missing

Reply:
ahh like a loading spinner at the bottom when we have content, and are loading more? can def add

Reply:
yep! we do that on mobile web currently (or at least we should), and definitely on desktop web